changelog/skip-ssl-running-tests-locally

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Skip SSL verification when running the PHP tests locally.
```

Comment:
This change is not needed to add to changelog for a new version, so we just need a comment here instead. This is possible by choosing `patch` then skipping a changelog entry with `npm run changelog`.

Similar to this: https://github.com/Automattic/woocommerce-payments/commit/9ac9f13dacb04a6c9cd9a0ded4cc2e3a30854cc1#diff-59b7da3c7ead90eae0a94dcf328f87d68c62c0926810c98ff7b132da63923a1b

Reply:
TIL! Thanks!

Reply:
Addressed here 22f81527465b07196dfda0341c02a540b52a4b87